### DISCUSSION TITLE
Celltype and condition specific GRNs

### DISCUSSION THREAD

**Discussion by ctspencer** [aertslab/SCENIC#452] [* General SCENIC questions]
I have a large dataset (>1 million cells) and am attempting to infer celltype and condition-specific GRNs. I ran the SCENIC pipeline after submitting the data by celltype and condition (case/control). I ran AUCell with the celltype/condition-specific regulons of a celltype known to be highly implicated in case samples; however, after Z-score normalization of auc_mtx I do not see the celltype/condition-specific regulons highly enriched in the celltype they come from. 

Is AUCell the best way to test for celltype regulon enrichment? 

Any suggestions/ideas on why regulons derived from a celltype are not enriched in  that celltype against other celltypes in the dataset?

**Comment by ZunpengLiu** [comment]
I have the same questions as well.  Wondering have you figured it out?
